Makefile

```
@@ -24,9 +24,24 @@ benchmark: ## Runs the benchmark suite
 	go test -bench=. -benchmem ./internal/repair/
 
 check: ## Runs all static analysis and tests on the application code
+	@$(MAKE) check-slop
 	@$(MAKE) lint
 	@$(MAKE) test
 
+check-slop: ## Checks relevant text files for punctuation used by AI
```

checkmake (0.3.2)

[warning] Target body for "check-slop" exceeds allowed length of 5 lines (12).

(maxbodylength)